packages/@core.keystone/cache.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Keystone-level cache.
+ *
+ * This package adds basic caching capabilities to keystone.
+ *
+ * Requirements:
+ * 1. Every request should have x-request-id. If it doesn't have it -- this request will not use cache
+ *
+ * Installation:
+ * 1. Have a custom keystone server
+ * 2. Somewhere in index.js call initCache(keystone, cache)
+ * 3. Add a middleware KeystoneLevelCache(cache) before keystone AdminApp or GraphQL app
```

Comment:
it's easier just add/remove the cache APP

Reply:
Done!